src/client/styles/objects/_reading-width.scss

```diff
@@ -1,3 +1,4 @@
 .reading-width {
   @include reading-width;
+  max-width: none !important;
```

Comment:
👍 

It's probably worth looking at the markup and understanding where this class is being used. Ideally it should only be added to elements that contain blocks of text. If it's sitting above a lot of elements, some of which need to be allowed to span the whole container, then it might be better to restructure the hierarchy and make sure this class is only used on elements where text appears.